packages/nitro-protocol/package.json

```diff
@@ -70,13 +69,13 @@
   "scripts": {
     "build:typescript": "rm -rf lib; yarn tsc -b .",
     "build:webpack": "webpack",
-    "contract:compile": "yarn node ./bin/compile.js",
+    "contract:compile": "yarn hardhat compile --force",
```

Comment:
do you think that the `force` flag is necessary?

Reply:
My instinct is that it is worth waiting a few seconds to get a bit more certainty that the compilation artifacts are up to date, especially as these are form part of the npm package. 

If developing with the contracts, one has the option to just run `yarn hardhat compile` and get a bit of a faster flow going. (On my machine it takes 2s to decide "nothing to compile" and 7s to compile everything.)

There's also the option to watch for file changes, and then recompile and/or redeploy as necessary https://hardhat.org/plugins/hardhat-watcher.html